ui/nuxeo-format-behavior.js

```diff
@@ -137,17 +137,29 @@ export const FormatBehavior = [
       if (!type) {
         return;
       }
-      const key = `label.document.type.${type.toLowerCase()}`;
-      const value = this.i18n(key);
-      return value === key ? type : value;
+      return this._getI18nWithPrefix('label.document.type', type.toLowerCase());
+    },
+
+    /**
+     * Returns the label for the given document permission
+     */
+    formatPermission(permission) {
+      const loweredKey = permission.substring(0, 1).toLowerCase() + permission.substring(1);
```

Comment:
We need to check that permission is defined otherwise we'll get a `Cannot read property 'substring' of undefined`.